contracts/external/proxies/SignedOperationProxy.sol

```diff
@@ -295,8 +296,8 @@ contract SignedOperationProxy is
                 authIdx
             );
 
-            // consider the signer to be msg.sender unless there is a signature
-            address signer = msg.sender;
+            // consider the signer to be null unless there is a signature
+            address signer = address(0);
```

Comment:
can you explain this change? this seems like a significant change

Reply:
https://github.com/dydxprotocol/solo/pull/303/files#diff-8104481c278ed60013172173fa1ec998R301

Reply:
ah right